app/controllers/pending_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PendingController < ApplicationController
+  def index
```

Comment:
hmm could this method just be a method in application controller? And then call @vendors?